browser/widevine/widevine_permission_request_browsertest.cc

```diff
@@ -260,6 +261,11 @@ class ScriptTriggerWidevinePermissionRequestBrowserTest
 
 IN_PROC_BROWSER_TEST_F(ScriptTriggerWidevinePermissionRequestBrowserTest,
                        SuggestPermissionIfWidevineDetected) {
```

Comment:
Shouldn't we disable test case like this?
```cpp
#if BUILDFLAG(IS_WIN) && defined(ARCH_CPU_X86)
#define MAYBE_SuggestPermissionIfWidevineDetected DISABLED_SuggestPermissionIfWidevineDetected
#else
#define MAYBE_SuggestPermissionIfWidevineDetected SuggestPermissionIfWidevineDetected
#endif
IN_PROC_BROWSER_TEST_F(Maybe_ScriptTriggerWidevinePermissionRequestBrowserTest,
   MAYBE_SuggestPermissionIfWidevineDetected) {
...
```

Reply:
Yeah, right. Thanks for spotting!